modular_skyrat/modules/gunsgalore/code/ammo/ammo.dm

```diff
@@ -63,3 +71,175 @@
+	desc = "A 5.6x40mm armor-piercing bullet casing."
+	extended_desc = "ARMOR PIERCING: Increased armor piercing capabilities. What did you expect?"
+	projectile_type = /obj/projectile/bullet/a762x39
+
+/obj/projectile/bullet/a762x39/ap
+	name = "5.6mm armor-piercing bullet"
+	damage = 30
+	wound_bonus = 15
+	armour_penetration = 60
+	wound_falloff_tile = 0
+
+/obj/item/ammo_casing/realistic/a762x39/emp
+	name = "5.6x40mm ion bullet casing"
+	desc = "A 5.6x40mm ion bullet casing."
+	extended_desc = "EMP: Produces an Electro-Magnetic Pulse on impact, damaging electronics severely."
+	projectile_type = /obj/projectile/bullet/a762x39/emp
+
+/obj/projectile/bullet/a762x39/emp
+	name = "5.6mm ion bullet"
+	damage = 25
+	wound_bonus = 15
+	armour_penetration = 15
+	var/emp_radius = 0
+
+/obj/projectile/bullet/a762x39/emp/on_hit(atom/target, blocked = FALSE)
+	..()
+	empulse(target, emp_radius, emp_radius)
+	return BULLET_ACT_HIT
+
+/obj/item/ammo_casing/realistic/a762x39/civilian
+	name = "5.6x40mm civilian bullet casing"
+	desc = "A 5.6x40mm civilian-grade surplus bullet casing."
+	extended_desc = "CIVILIAN: Non-military ammunition with a low powder load. Performs worse in every aspect in comparison to its military variant."
+	projectile_type = /obj/projectile/bullet/a762x39/civilian
+
+/obj/projectile/bullet/a762x39/civilian
+	name = "5.6mm surplus bullet"
+	damage = 30
+	wound_bonus = 15
+	armour_penetration = 10
+	wound_falloff_tile = 3
+
+/obj/item/ammo_casing/realistic/a762x39/civilian/rubber
+	name = "5.6x40mm rubber bullet casing"
+	desc = "A 5.6x40mm civilian-grade rubber bullet casing."
+	extended_desc = "RUBBER: Less than lethal ammo. Deals both stamina and brute damage."
+	projectile_type = /obj/projectile/bullet/a762x39/rubber
+	harmful = FALSE
+
+/obj/projectile/bullet/a762x39/rubber
+	name = "5.6mm rubber bullet"
+	damage = 12
+	armour_penetration = 10
+	stamina = 30
+	ricochets_max = 6
+	ricochet_incidence_leeway = 0
+	ricochet_chance = 130
+	ricochet_decay_damage = 0.7
+	shrapnel_type = null
+	sharpness = NONE
+	embedding = null
+	wound_bonus = -50
+
+/obj/item/ammo_casing/realistic/a762x39/civilian/hunting
+	name = "5.6x40mm hunting bullet casing"
+	desc = "A 5.6x40mm jacketed soft point bullet casing."
+	extended_desc = "HUNTING: Ammo purpose-built to deal more damage against simplemobs than other Humans."
+	projectile_type = /obj/projectile/bullet/a762x39/hunting
+
+/obj/projectile/bullet/a762x39/hunting
+	name = "5.6mm hunting bullet"
+	damage = 20
+	wound_bonus = 10
+	armour_penetration = 10
+	wound_falloff_tile = 3
+		/// Bonus force dealt against certain mobs
+	var/nemesis_bonus_force = 25
+		/// List of mobs we deal bonus damage to
+	var/list/nemesis_path = list(/mob/living/simple_animal)
+
+/obj/projectile/bullet/a762x39/hunting/prehit_pierce(mob/living/target, mob/living/carbon/human/user)
+	if(istype(target, nemesis_path))
+		damage += nemesis_bonus_force
+	.=..()
+
+/obj/projectile/bullet/a762x39/hunting/xeno
```

Comment:
doesn't need to exist, period.

Reply:
Yeah, it was made to be used against the previous xenos; I was already told by my colleague to remove it and simply wanted to be sure it's totally not needed. Will remove it after the MODsuits go through and I resolve all of the conflicts.